gcc.symbols.imp

```diff
@@ -169,5 +169,13 @@
   # string/ostream/istream type.
   { symbol: [ "std::char_traits", private, "<string>", public ] },
   { symbol: [ "std::char_traits", private, "<ostream>", public ] },
-  { symbol: [ "std::char_traits", private, "<istream>", public ] }
+  { symbol: [ "std::char_traits", private, "<istream>", public ] },
+
+  { symbol: [ "std::size_t", private, "<cstddef>", public ] },
+  { symbol: [ "std::size_t", private, "<cstdio>", public ] },
+  { symbol: [ "std::size_t", private, "<cstdlib>", public ] },
+  { symbol: [ "std::size_t", private, "<cstring>", public ] },
+  { symbol: [ "std::size_t", private, "<ctime>", public ] },
+  { symbol: [ "std::size_t", private, "<cuchar>", public ] },
+  { symbol: [ "std::size_t", private, "<cwchar>", public ] }
```

Comment:
Could you add a trailing comma here in the interest of keep the next patch smaller? Thanks.

Reply:
Actually, I think that's invalid in yaml/json, so that's probably a bad idea :)